index.html

```diff
@@ -47,6 +47,9 @@ <h2 id="header">typings</h2>
             <text> / </text>
             <span id="tc-240" onclick="setTimeCount(240)">240</span>
           </span>
+          <span id="custom">
+            <span style="border-bottom: 2px solid;">CUSTOM</span>
+          </span>

```

Comment:
This could be simplified
```suggestion
          <span id="custom" style="border-bottom: 2px solid;">
            CUSTOM
          </span>
```